modules/internal/String.chpl

```diff
@@ -695,13 +737,22 @@ module String {
   /*
     Creates a new string by creating a copy of the buffer of a `c_string`.
 
-    :arg s: Object to copy the buffer from
-    :type s: `c_string`
+    :arg x: Object to copy the buffer from
+    :type x: `c_string`
 
-    :arg length: Length of the `c_string` in bytes, excluding the terminating
-                 null byte.
+    :arg length: Length of the string stored in `x` in bytes, excluding the
+                 terminating null byte.
     :type length: `int`
 
+    :arg policy: `decodePolicy.strict` raises an error, `decodePolicy.replace`
```

Comment:
This text is taken from bytes.decode and has been there for a while. But I think you have a point that this can be a bit cleaner.

However, I still am a bit reluctant to make users do one more click to read that information. I am wondering if I can do a bullet list here (in the sense that whether chpldoc would be happy about that) and bytes.decode to make the list more readable. How does that sound?

Reply:
I was not aware that the text has been like that for awhile. If leaving things as is makes things easy for you I wouldn't mind it that much, but I definitely would find a bullet list easier to read (though like you I'm not sure if `chpldoc` will accept it).